conf/main/grakn.properties

```diff
@@ -109,6 +109,7 @@ loader.threads=0
 #Authentication
 password.protected=false
 JWT.secret=Anzdz9l4TG8N5y18oaTDueqJQca1aX7loaz0U8Hz
+admin.password=grakn
```

Comment:
We should not be storing passwords in plain text. Im not sure what an alternative would be in this case. Perhaps we store it in the system graph and remove access to that graph?